contrib/cirrus/cirrus_yaml_test.py

```diff
@@ -60,20 +60,20 @@ def test_depends(self):
                 msg=('No success aggregation task depends_on "{0}"'.format(task_name))
                 self.assertIn(task_name, success_deps, msg=msg)
 
-    def test_skips(self):
+    def test_only_if(self):
         """2024-06 PR#23030: ugly but necessary duplication in skip conditions. Prevent typos or unwanted changes."""
```

Comment:
not to self update PR number to this one for next push

Reply:
oops